app/Transformers/WalletTransformer.php

```diff
@@ -29,7 +33,21 @@ public function transform(Wallet $wallet)
             'visible' => $wallet->shown,
             'currency_code' => $wallet->currency_code,
             'order' => $wallet->order,
-            'created_at' => $wallet->created_at->toDateTimeString()
+            'created_at' => $wallet->created_at->toIso8601String()
         ];
     }
+
+    /**
+     * Includes transactions
```

Comment:
Try to formulate a better description. I know this is an obvious function but the function description here is the exact same as the function name.